FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -55,9 +38,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             let incomingURL = userActivity.webpageURL else {
             return false
         }
-        // sets a string like 'usps-postal-service-covid-funding' that we can use when issues are loaded
-        UserDefaults.standard.set(incomingURL.lastPathComponent, forKey: UserDefaultsKey.selectIssuePath.rawValue)
```

Comment:
we don't need this anymore?

Reply:
Nope, we get the url straight from `onOpenURL` and can do the same `lastPathComponent` parsing there